### ISSUE TITLE
cdn.mathjax.org end-of-life

### ISSUE THREAD

**Issue by pkra** [apendua/meteor-mathjax#15]
See https://www.mathjax.org/cdn-shutting-down/ for more details on migrating to another service.

**Comment by apendua** [comment]
@pkra Good catch! I wasn't aware of that. Luckily, `meteor-mathjax` can be configured to used a different download path by simply overwriting `MeteorMathJax.sourceUrl`. But of course, we should probably provide a new default value.
